solution/ui/regulations/eregs-vite/src/components/resources/SearchGovResults.vue

```diff
@@ -0,0 +1,168 @@
+<script setup>
+import {
+    formatDate,
+    locationLabel,
+    locationUrl,
+} from "../../utilities/filters";
+
+import ResultsItem from "@/components/search/ResultsItem.vue";
+import SupplementalContentObject from "eregsComponentLib/src/components/SupplementalContentObject.vue";
+
+defineProps({
+    base: {
+        type: String,
+        required: true,
+    },
+    partsLastUpdated: {
+        type: Object,
+        required: true,
+    },
+    partsList: {
+        type: Array,
+        required: true,
+    },
+    results: {
+        type: Array,
+        default: () => [],
+    },
+});
+
+const openChar = ""; // &#xe000;
+const closeChar = ""; // &#xe001;
```

Comment:
not sure how to target these characters via an explicit unicode/HTML code reference in JS.